packages/atomic/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "./tsconfig.stencil.json",
   "compilerOptions": {
+    "types": ["@testing-library/jest-dom"],
```

Comment:
This was required to avoid type collision and use the `toBeInTheDocument` method in    `expect(...).toBeInTheDocument()`